app/services/by_month_project_fetcher_service.rb

```diff
@@ -14,6 +14,19 @@ def sorted_openers(month, year)
     sort_by_conditions_met_and_name(projects)
   end
 
+  def confirmed_openers_by_team(month, year, team)
+    users = User.by_team(team)
+
+    projects = Conversion::Project.assigned_to_users(users).opening_by_month_year(month, year).includes(:tasks_data)
```

Comment:
If we add a `by_team` scope to Project could we avoid the extra DB query here? It is ultimately the projects that are in the same team as the current user and we have the `current_user`?

It would then be:

`Conversion::Project.by_team(current_user.team).opening_by_month_year(month, year).includes(:tasks_data)`

Reply:
*Is* a project always in the same team as the user, though? When we add a project and/or assign a user to a project, are we updating the project's team? I'm not convinced.

Reply:
Or are you talking about a project's *region* matching the user's team?

Reply:
Great point!! Let me think it over in the morning when my brain might be working!

Reply:
Right, brain freshley caffinated and working as well as can be expected! ☕ 

I think as ever, the answer to this is: it depends!

The logic is something like this:

```
if user.regional_casework_services_team?
  Projects.by_team(team)
else
  Projects.by_region(team)
end
```

The RCS team want to see:

> all projects that are assigned to them across all regions

A region team wants to see:

> all projects in their region across all teams

Reply:
I think I've fixed this 🤞